tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-panel.html

```diff
@@ -0,0 +1,256 @@
+<!--
+@license
+Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+<link rel="import" href="../paper-button/paper-button.html">
+<link rel="import" href="../paper-input/paper-input.html">
+<link rel="import" href="../paper-radio-button/paper-radio-button.html">
+<link rel="import" href="../paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../tf-storage/tf-storage.html">
+
+<!--
+  Setup dialog for what-if tool.
+-->
+<dom-module id="tf-inference-panel">
+  <template>
+    <style id="linter-paper-button-style">
+      /**
+       * This style preserves the styling previous to
+       * https://github.com/PolymerElements/paper-button/pull/115
+       * This change can break the layout of paper-button content.
+       * Remove this style to apply the change, more details at b/70528356.
+       */
+      paper-button {
+        display: inline-block;
+        text-align: center;
+        font-family: inherit;
+      }
+    </style>
+    <style>
+      :host {
+        display: block;
+        font-size: 14px;
+      }
+      tf-option-selector .model-type-button {
+        width: auto;
+      }
+      .flex-holder {
+        display: flex;
+      }
+      .title {
+        font-size: 32px;
+        margin-bottom: 20px;
+      }
+      .help-icon {
+        width: 15px;
+        height: 15px;
+        padding: 0px;
```

Comment:
super nit: please omit unit for `0` values as per Google css style guide.
https://google.github.io/styleguide/htmlcssguide.html#0_and_Units

Reply:
done